pkg/kbld/config/config.go

```diff
@@ -157,16 +158,14 @@ func NewConfigFromImagesLock(res ctlres.Resource) (Config, error) {
 		return Config{}, err
 	}
 
-	var imagesLock ImagesLock
-
-	err = yaml.Unmarshal(iLockBytes, &imagesLock)
+	imagesLock, err := lockconfig.NewImagesLockFromBytes(iLockBytes)
 	if err != nil {
-		return Config{}, fmt.Errorf("Unmarshaling %s: %s", res.Description(), err)
+		return Config{}, fmt.Errorf("load ImagesLock file: %s", err)
```

Comment:
i think we should still have something like `return Config{}, fmt.Errorf("Unmarshaling %s as ImagesLock: %s", res.Description(), err)` that includes `res.Description()`